### PR TITLE
MINOR FIX: Course Validation Invalid Status 

### PR DESCRIPTION
**Pull request by tsatay**

### CODE REVIEW

**Comment by hassanhabib** [comment]
@tsatay we need to make tests are passing here.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.cs [hunk 1]
```diff
@@ -86,6 +86,19 @@ public static IEnumerable<object[]> InvalidMinuteCases()
...
         }
 
+        private static T GetInvalidEnum<T>()
+        {
+            int randomNumber = GetRandomNumber();
+            while(Enum.IsDefined(typeof(T), randomNumber) is true)
+            {
+                randomNumber = GetLocalRandomNumber();
+            }
+            return (T)(object)randomNumber;
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.cs:96]: Just add one line above this one.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.cs [hunk 2]
```diff
@@ -86,6 +86,19 @@ public static IEnumerable<object[]> InvalidMinuteCases()
             };
         }
 
+        private static T GetInvalidEnum<T>()
+        {
+            int randomNumber = GetRandomNumber();
+            while(Enum.IsDefined(typeof(T), randomNumber) is true)
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.cs:92]: Add just one line above this one.

**Comment by ShriHumrudha** [comment]
@tsatay , need your attention here.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.cs (hunk 1 above)
Comment (tsatay) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.cs:98]: It's fixed.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.cs (hunk 2 above)
Comment (tsatay) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.cs:93]: it's fixed

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.Validations.Add.cs [hunk 3]
```diff
@@ -48,6 +48,51 @@ public async void ShouldThrowValidationExceptionOnCreateWhenCourseIsNullAndLogIt
...
         }
 
+        [Fact]
+        public async void ShouldThrowValidationExceptionOnCreateIfCourseStatusIsInvalidAndLogItAsync()
+        {
+            DateTimeOffset randomDateTime = GetRandomDateTime();
+            Course randomCourse = CreateRandomCourse(randomDateTime);
+            Course invalidCourse = randomCourse;
+            invalidCourse.Status = GetInvalidEnum<CourseStatus>();
+
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.Validations.Add.cs:58]: This line needs to be removed.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.Validations.Add.cs (hunk 3 above)
Comment (tsatay) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Courses/CourseServiceTests.Validations.Add.cs]: It's fixed

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**